src/irclog2html/irclogsearch.py

```diff
@@ -52,12 +45,6 @@
 from .logs2html import find_log_files
 
 
-try:
-    unicode
-except NameError:
-    # Python 3.x
-    unicode = str
-
 
```

Comment:
flake8 says

> src/irclog2html/irclogsearch.py:49:1: E303 too many blank lines (3)

(and I cannot stop thinking that there ought to be a way to have GitHub Actions run a linter and annotate the actual diff with the comments, but I've never seen it done for a Python project.)

Reply:
I should investigate these two:

- https://github.com/marketplace/actions/flake8-action
- https://github.com/python-formate/flake8-github-actions?tab=readme-ov-file#use-with-github-actions